napi_from_js.md

```diff
@@ -0,0 +1,204 @@
+# N-API Function APIs 
+
+N-API provides a set of APIs that allow JavaScript code to 
+call back into native code. N-API APIs that support calling back
+into JavaScript take in callback function represented by 
+the `napi_callback` type. When the JavaScript VM calls back to
```

Comment:
I think you meant to write "APIs that support calling back into native code take in a callback function..."